### DISCUSSION TITLE
Floating wind turbine

### DISCUSSION THREAD

**Discussion by Nakhchi** [xcompact3d/Incompact3d#253] [Q&A] [answered]
Hi everyone,

Is it possible to add floating motion to wind turbines in ALM/ADM methods? If code modification is needed, your support is much appreciated. 

Many thanks,
Mahdi

**Comment by slaizet** [comment, marked as answer]
Hi,
This is not an option in the code. It would require huge developments that we are not planing to do anytime soon.
Best,
Sylvain
